src/cdsetool/download.py

```diff
@@ -97,6 +99,10 @@ def download_feature(
                         log.warning(e)
                         continue
                 # Close file before copy so all buffers are flushed.
+                if checksum:
+                    checksum_for_file(tmp_file, title, options.get("proxies", None))
+                    os.remove(tmp_file)
+                    continue  # download retry
```

Comment:
I've had plenty of archives containing files with bad checksums from Copernicus, but it's always been the source archives that have been faulty, so downloading them again will just give the same error again.

Reply:
Yeah I know that that is a big problem, from Copernicus not providing reliable metadata. I guess you can just bypass the checksum by giving it in the parameters for download to false  (`{'checksum': False}`). Maybe I can put the default checksum value to false and then if someone needs it can be change it to true to use it. Even though this logic has to be switched in the future  since in a working environment where the checksum provided are valid we should always check if the products where downloaded correctly. Let me know if you prefer the checksum disabled from the beginning or not